mitosheet/src/components/endo/ColumnHeaderDropdown.tsx

```diff
@@ -96,6 +103,36 @@ export default function ColumnHeaderDropdown(props: {
                 tooltip={!isNumberDtype(props.columnDtype) ? "Only number columns can be formatted currently" : undefined}
             />
             <DropdownSectionSeperator isDropdownSectionSeperator/>
+            <DropdownItem 
+                title='Set Column Formula'
+                onClick={() => {
+                    const columnFormula = getCellDataFromCellIndexes(props.sheetData, rowIndex, columnIndex).columnFormula;
+
+                    props.setEditorState({
+                        rowIndex: 0,
+                        columnIndex: columnIndex,
+                        formula: columnFormula !== undefined ? columnFormula : '',
+                        // As in google sheets, if the starting formula is non empty, we default to the 
+                        // arrow keys scrolling in the editor
+                        arrowKeysScrollInFormula: columnFormula !== undefined && columnFormula.length > 0,
+                        editorLocation: 'cell'
+                    })
+                }}
+                disabled={columnFormula === undefined || columnFormula.length == 0}
+                tooltip={!isNumberDtype(props.columnDtype) ? "Data columns don't support formulas" : undefined}
```

Comment:
I don't think this is the right check! I think you want the disabled check above!